yarpl/test/FlowableTest.cpp

```diff
@@ -16,7 +16,8 @@ void unreachable() {
 template <typename T>
 class CollectingSubscriber : public Subscriber<T> {
  public:
-  explicit CollectingSubscriber(int64_t requestCount = 100)
```

Comment:
We'd have to at least add `std::vector<T>& TestSubscriber::values();` and a ctor that controls the initial request count, but yeah I can see they're more or less the same.